src/js/jsx/sections/libraries/assets/CharacterStyle.jsx

```diff
@@ -84,21 +86,20 @@ define(function (require, exports, module) {
                 fontColorHex = tinycolor(color.value).toHexString().toUpperCase();
             }
 
-            var fontSizeAndColorStr = _.remove([fontSizeStr, fontColorHex], null).join(", ");
-
-            var classNames = classnames("libraries__asset", {
-                "assets__graphic__dragging": this.props.isDragging,
-                "libraries__asset-selected": this.props.selected
-            });
-
-            var displayName = element.displayName !== "" ? element.displayName : font.name + " " + font.style,
+            var fontSizeAndColorStr = _.remove([fontSizeStr, fontColorHex], null).join(", "),
+                displayName = element.displayName !== "" ? element.displayName : font.name + " " + font.style,
```

Comment:
This is confusing to me, is it either the display name, or font name + " " + style...

or is it either the display name or the font name , followed by space and font style. Can you add parentheses to make it clear.